### ISSUE TITLE
remove filtering for departments in household travel surveys

### ISSUE THREAD

**Issue by sebhoerl** [eqasim-org/ile-de-france#239]
When introducing the option to filter for urban types (unités urbaines) the idea was to at the same time remove the functionality that filters household travel surveys to the requested departments. The idea that this is done manually, if required, in the statistical matching algorithm. So the HTS stages should be adjusted such that no filtering happens (especially ENTD).

**Comment by vincent-leblond** [comment]
Hi @sebhoerl,
On some departments, this issue is very important. And even essential. Is there any one working on it ? If not, Marie can make a PR which is almost already prepared. This can be done next week.
Your commit #247 is great as it should fix a problem that I have also found on one specify person in ENTD sample. I had to remove this person from the sample in order to use the whole ENTD sample.
